tests/test_user/test_login.py

```diff
@@ -0,0 +1,57 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from django.contrib.auth.models import User
+
+
+class AccountsTestCase(TestCase):
+    def setUp(self):
+        user = {
+            'username': 'demo_user'
+        }
+        User.objects.create_user(
```

Comment:
Please go through this coding style guide https://docs.djangoproject.com/en/2.1/internals/contributing/writing-code/coding-style/ and change this line break according to it.

Reply:
done